cmd/legacy/configread/configread.go

```diff
@@ -27,7 +27,7 @@ func Run(v *viper.Viper) (int, error) {
 		)
 	}
 
-	fmt.Println(output)
+	fmt.Print(output)
```

Comment:
won't this cause the output to be joined with the terminal prompt without a newline?

Reply:
We actually don't need to change it. Even for last line where there's no newline it will print accorndingly.We actually don't need to change it. Even for last line where there's no newline it will print accorndingly.